src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,17 @@
+(ns status-im2.contexts.wallet.events
+  (:require [utils.re-frame :as rf]
+            [taoensso.timbre :as log]))
+
+(rf/defn add-chains-to-db
+  {:events [:wallet-2/add-chains-to-db]}
```

Comment:
Maybe a better name is `:wallet-2/get-ethereum-chains` because that's exactly what the RPC endpoint does. I think it's redundant to specify in the event ID if anything is stored in the app DB because events should always be free to manipulate the app DB, that's essentially why they exist.